dev/ProjectReunion_DLL/ProjectReunion_DLL.vcxproj

```diff
@@ -258,6 +253,7 @@
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <AdditionalOptions>/ZH:SHA_256 %(AdditionalOptions)</AdditionalOptions>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
The other thread about linker configuration settled on a "we should do this for the build pipeline but not developer desktop runs."  Does CFG add cost or build time, or prevent incremental linking?

Reply:
CFG doesn't affect incremental linking. It is, however, incompatible with /Zi (but not /ZI) which is why I didn't turn it on for Debug builds.

I built with CFG on and off several times with Visual Studio's build timing enabled, and I couldn't discern a meaningful difference. CFG adds a couple intructions around indirect calls (like virtual dispatch, function pointers), so it added 5 kB to Microsoft.ProjectReunion.dll (385 kB vs 380 kB). /ZH:SHA_256 may have added a **tiny** amount of compile time, but it was so low as to be nearly indistinguishable from random noise (~100ms out of ~16 seconds).

The impact on inner-loop dev experience seems to be somewhere between "negligible" and "tiny", so it's worth keeping the pipeline and local build settings in sync.